File: exercicio02/python/fazenda.py
class Fazenda:
    nome = ""
    area = 0.0
    propietario = ""

    def __init__(self, nome: str, area: float, propietario: str):
        self.nome = nome
        self.area = area
        self.propietario = propietario

    def imprimi(self):
        print("Fazenda:", self.nome)
        print("Área:", self.area, "a")
        print("Propietário:", self.propietario)

File: exercicio02/java/Fazenda.java
package exercicio02.java;

public class Fazenda {
  public String nome;
  public double area;
  public String propietario;

  public Fazenda(String nome, double area, String propietario) {
    this.nome = nome;
    this.area = area;
    this.propietario = propietario;
  }

  public void imprimi() {
    System.out.println("Fazenda: " + this.nome);
    System.out.println("Area: " + this.area + "hectares");
    System.out.println("Proprietario: " + this.propietario);
  }
}
